src/style.css

```diff
@@ -0,0 +1,137 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+html {
+  margin: 0;
+
+}
+
+.container {
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  flex-direction: column;
+}
+
+.container__card {
+  height: 134px;
+}
+
+.container__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 3px;
+}
+
+.container__text {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  margin-top: 40px;
+  box-sizing: border-box;
+  display: flex;
+
+}
+
+.container__code {
+  width: 92px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  box-sizing: border-box;
+  margin-top: 4px;
+  color: #616070;
+}
+
+.container__star {
+  margin-top: 16px;
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 span:nth-child(-n+4) {
```

Comment:
Please, look at checklist (item 3). You need use other classes, as in task "Stars".